.travis.yml

```diff
@@ -17,8 +17,8 @@ after_script: greenkeeper-lockfile-upload
 
 after_success:
   - bash <(curl -s https://codecov.io/bash)
-  - yarn global add --ignore-engines semantic-release@11
-  - semantic-release
+  - yarn global add --ignore-engines travis-deploy-once@4 semantic-release@12
```

Comment:
Why install `semantic-release` if it might not even be used(like node 4)? Please change it to what I had suggested in https://github.com/okonet/lint-staged/pull/371#discussion_r159149053. This will allow us to drop `--ignore-engines` as well.

Reply:
I did this to match their migration guide and also to make the `yarn global add semantic-release` process not take part in `travis-deploy-once`'s job.

Will there be a negative effect of doing this?

Reply:
There isn't anything new that I can add to answer that question. Like I mentioned, 
1. We are unnecessarily installing something we won't need on all node versions except 8.
2. We are forced to install using `--ignore-engines` which I'd really like to avoid since there is a pretty simple way to do so.

Reply:
> 1. We are unnecessarily installing **something we won't need on all node versions** except 8.
> 2. We are forced to install using **--ignore-engines which I'd really like to avoid** since there is a pretty simple way to do so.

Well, all I can say is you're worrying something that you should not. The thing that we're discussing here has no effects on our development, and even it will never bother our end users. `semantic-release` is a tool to help us in automatic publishing, right? 😄 

I agree with you that it's an ugly thing to do the installation with `--ignore-engines` **but only** if we're doing it to install our devDeps and direct dependencies because it'll most likely cause problems for our users. But it's not the case for a tool that will never affect our users, like `semantic-release`.

But yeah, whatever it is, as long as it'll still make `semantic-release` works fine, I'm okay.
It's not a big deal though. Will have no effect for `lint-staged` itself.

Reply:
Ha ha.. Not worried mate. It's an honest review with very clear pointers for what I think could be improved. Feel free to dismiss it if you disagree. But for what it's worth, I don't think I am being illogical and I haven't seen anything that would help me change my mind.